.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules/
 artifacts/
-build/
+**/gen/**/build/
```

Comment:
I don't think we should ignore this file. I would say this is fine if this was just about a text file with a value but we have some Go code there. I think it's better if we don't add it to `.gitignore` even if it shows in the local diff occasionally.

Reply:
Sorry, I'm not following. I've changed `build/` to `**/gen/**/build/` because the newly added package `build` that handles versioning now (`build.go`) was ignored and CI was failing. `build/` was ignoring every `build` folder. We have a bunch of files under `**/gen/**/build/` that probably should still be ignored.

Reply:
Ah, my fault!

Reply:
Looks like generated abis. Here's a path where you can find those files `keep-core/pkg/chain/ethereum/beacon/gen/_contracts/package/build/contracts`